doc/internal/ReleaseInstructions.md

```diff
@@ -214,17 +214,23 @@ Create the `settings.xml` in the `$HOME/.m2/` directory as described in their [i
 ### Creating Release (or Candidate)
 
 1. Create a new branch (if needed) with the following existing naming convention e.g. release-9.0. We usually do this while creating RC1 and re-use it for GA release.
+![GitHub Release01](/.images/release-01.png)
 2. Click on Code -> Releases
+![GitHub Release02](/.images/release-02.png)
 3. Draft a new release. The naming convention for release candidates is to append “-rc1” to the expected release version.
+![GitHub Release03](/.images/release-03.png)
 4. Tag a new release against the branch.
+![GitHub Release04](/.images/release-04.png)
 5. Copy/paste previously built Release Notes into the description.
+![GitHub Release05](/.images/release-05.png)
 
 Note: If this is a pre-release select the pre-release checkbox. 
 
 ### Post Release Steps
 * Announce new release in Vitess Slack #general channel. 
 * Create a new [PR](https://github.com/vitessio/website/pull/670) for Vitess Blog.
```

Comment:
PR should be created and approved pre-release so that as soon as release is out we can merge and  the blog post.